File: lambda/src/main/scala/com/spacecorpshandbook/ostium/lambda/proxy/ApiGatewayProxyResponse.scala
package com.spacecorpshandbook.ostium.lambda.proxy

import java.util.Map

import com.spacecorpshandbook.ostium.core.model.CancelResponse

import scala.beans.BeanProperty

/**
  * Required response structure for the AWS API Gateway
  */
class ApiGatewayProxyResponse {

  @BeanProperty
  var statusCode: String = _

  @BeanProperty
  var headers: Map[String, String] = _

  @BeanProperty
  var body: String = _

}

File: lambda/src/main/java/temp/ApiGatewayProxyResponse.java
package temp;

import java.util.Map;

/**
 * Created by justin on 12/8/16.
 */
public class ApiGatewayProxyResponse {

    private String statusCode;

    private Map<String, String> headers;

    private String body;

    public String getStatusCode() {
        return statusCode;
    }

    public void setStatusCode(String statusCode) {
        this.statusCode = statusCode;
    }

    public Map<String, String> getHeaders() {
        return headers;
    }

    public void setHeaders(Map<String, String> headers) {
        this.headers = headers;
    }

    public String getBody() {
        return body;
    }

    public void setBody(String body) {
        this.body = body;
    }
}
